tests/Feature/CreateDatabaseTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Actions\CreateDatabase;
+use App\Shell\Shell;
+use Illuminate\Support\Facades\Config;
+use Symfony\Component\Process\ExecutableFinder;
+use Tests\Feature\Fakes\FakeProcess;
+use Tests\TestCase;
+
+class CreateDatabaseTest extends TestCase
+{
+    /** @test */
+    public function it_creates_a_mysql_database()
+    {
+        $this->fakeLamboConsole();
+
+        Config::set('lambo.store.create_database', true);
+        Config::set('lambo.store.database_username', 'user');
+        Config::set('lambo.store.database_password', 'password');
+        Config::set('lambo.store.database_name', 'database_name');
+
+        $this->mock(Shell::class, function ($shell) {
+            $shell->shouldReceive('execInProject')
+                ->with('mysql --user=user --password=password -e "CREATE DATABASE IF NOT EXISTS database_name";')
+                ->once()
+                ->andReturn(FakeProcess::success());
+        });
+        $this->assertEquals('Created a new database ' . Config::get('lambo.store.database_name'), app(CreateDatabase::class)());
```

Comment:
All of these feel a bit fragile, since the moment we change the message they'll fail. Maybe we just check for the database name in the message, and assert the call was actually made to shell or something?

Reply:
Done.